src/activitypub/fediverse-bridge.ts

```diff
@@ -87,81 +82,15 @@ export class FediverseBridge {
         if (!post.author.isInternal) {
             return;
         }
-        const ctx = this.fedifyContextFactory.getFedifyContext();
-        let fedifyObject: FedifyNote | Article;
-
-        let mentions: Mention[] = [];
-        let ccs: URL[] = [];
-
-        if (post.type === PostType.Note) {
-            if (post.inReplyTo) {
-                return;
-            }
-            mentions = post.mentions.map(
-                (account) =>
-                    new Mention({
-                        name: `@${account.username}@${account.apId.hostname}`,
-                        href: account.apId,
-                    }),
-            );
-            ccs = [
-                post.author.apFollowers,
-                ...mentions.map((mention) => mention.href),
-            ].filter((url) => url !== null);
-
-            fedifyObject = new FedifyNote({
-                id: post.apId,
-                attribution: post.author.apId,
-                content: post.content,
-                summary: post.summary,
-                published: Temporal.Now.instant(),
-                attachments: post.attachments
-                    ? post.attachments
-                          .filter((attachment) => attachment.type === 'Image')
-                          .map(
-                              (attachment) =>
-                                  new Image({
-                                      url: attachment.url,
-                                  }),
-                          )
-                    : undefined,
-                tags: mentions,
-                to: PUBLIC_COLLECTION,
-                ccs: ccs,
-            });
-        } else if (post.type === PostType.Article) {
-            const preview = new FedifyNote({
-                id: ctx.getObjectUri(FedifyNote, { id: String(post.id) }),
-                content: post.excerpt,
-            });
-            ccs = post.author.apFollowers ? [post.author.apFollowers] : [];
-
-            fedifyObject = new Article({
-                id: post.apId,
-                attribution: post.author.apId,
-                name: post.title,
-                summary: post.summary,
-                content: post.content,
-                image: post.imageUrl,
-                published: Temporal.Instant.from(
-                    post.publishedAt.toISOString(),
-                ),
-                preview,
-                url: post.url,
-                to: PUBLIC_COLLECTION,
-                ccs: ccs,
-            });
-        } else {
-            throw new Error(`Unsupported post type: ${post.type}`);
+
+        if (post.type === PostType.Note && post.inReplyTo) {
```

Comment:
Maybe a TODO comment above this explaining that replies are handled in the handler, but they should be moved here eventually?